projects/dashboards/nfdi-gdo/src/app/species-selector/species-selector.component.html

```diff
@@ -98,7 +79,11 @@
     <div *ngIf="fishSpeciesLayer">
         <div class="spacer"></div>
         <geoengine-layer-list-element [layer]="fishSpeciesLayer" [menu]="false"></geoengine-layer-list-element>
-        <p class="mat-caption">Brunken, H. & Vatterrott, H.-R. (2020): Fischartenatlas von Deutschland und Österreich</p>
+        <p class="mat-caption">
+            Die hier zu Demonstrationszwecken gezeigten Daten stammen aus Brunken, H. & Vatterrott, H.-R. (2020): Fischartenatlas von
+            Deutschland und Österreich und Monitoringdaten, die für die Aktualisierung der Rote Liste der sich im Süßwasser reproduzierenden
```

Comment:
vielleicht den Publikationstitel in Anführungszeichen setzen oder anders hervorheben